proto/greptime/v1/meta/heartbeat.proto

```diff
@@ -84,6 +87,11 @@ message RegionStat {
   map<string, bytes> extensions = 99;
 }
 
+message FlowStat {
+  // Each flow's in mem state's size in bytes
+  map<uint64, uint64> flow_stat_size = 1;
```

Comment:
I don't remember if we defined a type for FlowId

Reply:
> FlowId

protoc have limit in key type, error msg will be `"protoc failed: greptime/v1/meta/heartbeat.proto:93:3: Key in map fields cannot be float/double, bytes or message types.\n"` if use FlowId, should I just use a uint32 instead?